modules/machineset-yaml-nutanix.adoc

```diff
@@ -78,26 +78,31 @@ endif::infra[]
       providerSpec:
         value:
           apiVersion: machine.openshift.io/v1
-          cluster:
+          categories: <5>
+            key: <prism_category_name>
+            value: <prism_category_value>
+          cluster: <6>
             type: uuid
             uuid: <cluster_uuid>
           credentialsSecret:
             name: nutanix-creds-secret
           image:
-            name: <infrastructure_id>-rhcos <5>
+            name: <infrastructure_id>-rhcos <7>
             type: name
           kind: NutanixMachineProviderConfig
-          memorySize: 16Gi <6>
+          memorySize: 16Gi <8>
+          project: <9>
+            type: ""
```

Comment:
project is a resource identifier similar to images, subnets, and clusters and is either
```
- type: uuid
  uuid: project_uuid
```
or
```
- type: name
  name: project_name